tests/testthat/test-commas_linter.R

```diff
@@ -43,4 +43,8 @@ test_that("returns the correct linting", {
   expect_lint("a[1, , 2, , 3]",
     NULL,
     commas_linter)
+
+  expect_lint("purrr::partial(list, 1, ... = , 2)",
```

Comment:
Could you move this test into `test-error.R` as an example of some valid R code that has a (seemingly) incomplete equal-assignment statement. And rewrite it so that it tests that a parsing error does not throw:

```
# no parsing error is expected for the equals-assignment in this code:
expect_lint("purrr::partial(list, 1, ... = , 2)", NULL, equals_na_linter)
```

Reply:
Sure